projects/plugins/boost/app/assets/src/js/features/minify-meta/minify-meta.tsx

```diff
@@ -65,6 +79,11 @@ const MetaComponent = ( { buttonText, placeholder, datasyncKey }: Props ) => {
 
 	function loadDefaultValue() {
 		setInputValue( defaultValue );
+		setNotice( {
+			id: 'minify_' + concatenateType + '_load_default',
+			type: 'success',
+			message: __( 'Default handles loaded', 'jetpack-boost' ),
+		} );
```

Comment:
Same concern as loading default cornerstone pages.

Reply:
And removed too.